AuthServ/AuthServer.cpp

```diff
@@ -77,8 +77,14 @@ void auth_init(AuthServer_Private& client)
         client.m_buffer.writeBytes(serverSeed, 7);
     }
 
-    /* send reply */
+    /* send encryption reply */
     DS::SendBuffer(client.m_sock, client.m_buffer.buffer(), client.m_buffer.size());
+
+    /* inform client of server caps as part of the init process */
+    START_REPLY(e_AuthToCli_ServerCaps);
+    uint32_t caps = e_CapsScoreLeaderBoards;
```

Comment:
The reason this value is stored as a temporary is for future configurable capabilities, such as the planned cap for the Gahreesen wall.